package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@typescript-eslint/eslint-plugin": "^5.55.0",
     "@typescript-eslint/parser": "^5.55.0",
+    "axios": "^1.3.5",
```

Comment:
is it possible just use `fetch`? no 3rd party dependencies

Reply:
the thing is that the sdk might be used in browser and axios have wide browser support with other features like cancel request, timeout, interceptors, XSRF protection ...etc